tests/Unit/PointwiseFunctions/AnalyticSolutions/ScalarAdvection/Test_Krivodonova.cpp

```diff
@@ -0,0 +1,117 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Framework/TestingFramework.hpp"
+
+#include <random>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Evolution/Systems/ScalarAdvection/Tags.hpp"
+#include "Framework/CheckWithRandomValues.hpp"
+#include "Framework/SetupLocalPythonEnvironment.hpp"
+#include "Framework/TestCreation.hpp"
+#include "Framework/TestHelpers.hpp"
+#include "Helpers/DataStructures/MakeWithRandomValues.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/ScalarAdvection/Krivodonova.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/TaggedTuple.hpp"
+
+namespace {
+
+void test_create() noexcept {
+  const auto sol =
+      TestHelpers::test_creation<ScalarAdvection::Solutions::Krivodonova>("");
+  CHECK(sol == ScalarAdvection::Solutions::Krivodonova());
+}
+
+void test_serialize() noexcept {
+  ScalarAdvection::Solutions::Krivodonova sol;
+  test_serialization(sol);
+}
+
+void test_move() noexcept {
+  ScalarAdvection::Solutions::Krivodonova sol;
+  ScalarAdvection::Solutions::Krivodonova sol_copy;
+  test_move_semantics(std::move(sol), sol_copy);  //  NOLINT
+}
+
+struct KrivodonovaProxy : ScalarAdvection::Solutions::Krivodonova {
+  using ScalarAdvection::Solutions::Krivodonova::Krivodonova;
```

Comment:
What exactly does this line do?

Reply:
It "imports" the constructors of the superclass